Lesson_204V2/FullSolution/.gitignore

```diff
@@ -0,0 +1,212 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Is there a reason to provide a gitignore here? shouldn't the gitignore at the base of repo cover this?
(same with gitattributes above)